src/v2/util/FactorUtil.js

```diff
@@ -31,9 +31,16 @@ const factorData = {
     iconClassName: 'mfa-okta-call',
   },
 
+  // Will get rid of this after fully implementing verify. OKTA-301557
   'webauthn': {
```

Comment:
I don't fully understand why we are keeping this.. Was this added for LEA?

Reply:
It was added not for LEA but for Oktane demo, I think?